galaxy/tools/deps/mulled/mulled_build.py

```diff
@@ -144,6 +145,15 @@ def mull_targets(
         '-set', "BINDS='%s'" % bind_str,
         command,
     ]
+    if test_files:
+        test_bind = []
+        for test_file in test_files:
+            if ':' not in test_file:
```

Comment:
Yep, that's a good idea, docker would otherwise create a volume with that name. I think we can just silently skip those, the test should fail anyway with a pretty clear file not found error.